### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by devhell** [jacoborus/tender.vim#51]
Just wanted to say thank you for this colorscheme. I've been looking for the perfect colorscheme for many many years and tried pretty much everything, and no colorscheme has been as satisfying and fun to work with as this one.

I've been planning to work on creating tender colors for:
  - `profanity` (nearly done with this one actually)
  - `ncmpcpp`
  - `bat`
  - `neomutt`

Progress has been unfortunately very slow, but I'm also always looking out in case someone already made one for any of those programs. Anyway, just wanted to say thank you! :beer:

**Comment by ghost** [comment]
That's the best colorscheme, so clean and balanced

**Comment by simonward87** [comment]
Agree, love this theme. For `bat` try using the `base16` theme if you already have a tender terminal theme setup, works pretty well for me (just passes the terminal colours through).
